src/modules/core/components/Comment/Comment.tsx

```diff
@@ -136,7 +137,7 @@ const Comment = ({
             )}
         </div>
         <div className={styles.text} data-test="comment">
-          <Decorate>{comment}</Decorate>
+          <Decorate>{comment ? parse(comment) : comment}</Decorate>
```

Comment:
If the comment is falsy, like `undefined` or `''`, then shouldn't we just not render these tags?

Reply:
I was trying to keep the behavior similar to the previous behavior in the falsy case. But sure, I changed it to not render in the falsy case.